filters/replicateto/replicateto.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2015 Zalando SE
```

Comment:
please don't use the license text in the code files. The reason for many files having it, is that initially we thought that it is required. Turned out no, and since then when changing a file, we remove them, and when new ones are added, we don't add the license excerpt.